ambry-tools/src/main/java/com/github/ambry/cloud/azure/AzureCompactionTool.java

```diff
@@ -122,10 +122,10 @@ public static void main(String[] args) throws Exception {
    * PartitionId implementation that returns only its path.
    */
   private static class PartitionPathId implements PartitionId {
```

Comment:
nit: update the javadocs (because it doesn't return only path now).

Reply:
Done